netty/src/test/java/io/grpc/netty/NettyChannelBuilderTest.java

```diff
@@ -90,6 +93,24 @@ public void failInvalidAuthority() {
   }
 
   @Test
+  public void sslContextCanBeNull() {
+    NettyChannelBuilder builder = mock(NettyChannelBuilder.class);
+    builder.sslContext(null);
+  }
+
+  @Test
+  public void failIfSslContextIsNotClient() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("Server SSL context can not be used for client channel");
+
+    SslContext sslContext = mock(SslContext.class);
+    when(sslContext.isClient()).thenReturn(false);
```

Comment:
FYI: mocks return default values, and in this case, returning false is the default.